package.json

```diff
@@ -7,20 +7,26 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "fix-lint": "next lint --fix",
     "test": "jest --watchAll"
   },
   "dependencies": {
     "eslint": "8.45.0",
-    "eslint-config-next": "13.4.10",
+    "mongoose": "^7.4.1",
     "next": "13.4.10",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^6.0.4"
+    "styled-components": "^6.0.4",
+    "swr": "^2.2.0",
+    "uid": "^2.0.2",
+    "usestate": "^1.1.3",
```

Comment:
You don't need the package `usestate`.

Reply:
This is not the React `useState`. It appears that it's this [package](https://snyk.io/advisor/npm-package/usestate), on [npm](https://www.npmjs.com/package/usestate) here.